app/backend/src/createusers.py

```diff
@@ -0,0 +1,89 @@
+import csv
+import os
+import sys
+import math
+import random
+import logging
+import faker
+import multiprocessing
+from couchers.models import (
+    User,
+)
+from couchers import config
+from couchers.crypto import hash_password
+from couchers.servicers.api import hostingstatus2sql
+from couchers.utils import create_coordinate
+from couchers.db import session_scope
+
+fake = faker.Faker()
+
+logger = logging.getLogger(__name__)
+
+cities = []
+countries = set()
+world_population = 0
+
+# data from https://simplemaps.com/data/world-cities
+file_name = "src/data/worldcities.csv"
```

Comment:
You could use `os.path.dirname(__file__)` to make this path independent of the current working directory.

Great that you specify the source.